### PR TITLE
[issue-650] functionality split

### LINKED ISSUE #650: separate the package to /tabular and /core namespaced packages

### PR DESCRIPTION
**Pull request by yromanyshyn**
resolves #650

### CODE REVIEW

File: deepchecks/tabular/__init__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Package for tabular functionality."""
+from .dataset import Dataset
+from .base import (
+    TabularContext,
+    TabularCheck,
+    Suite,
```
Comment (matanper) [inline deepchecks/tabular/__init__.py:16]: I think if leave it "Suite" and not "TabularSuite" we can do the same for the rest of classes

Comment (yromanyshyn) [reply deepchecks/tabular/__init__.py:16]: like this?
- `TabularContext` -> `Context`
- `TabularCheck` -> `Check`

do not know/ @ItayGabbay , @noamzbr  what do you think?

Comment (ItayGabbay) [reply deepchecks/tabular/__init__.py:16]: I think it's better

Comment (yromanyshyn) [reply deepchecks/tabular/__init__.py:16]: renamed

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**